tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -62,15 +62,17 @@ func setUpTestDir() error {
 	return nil
 }
 
-func mountGcsfuse() error {
+func mountGcsfuse(flag string) error {
 	mountCmd := exec.Command(
 		binFile,
 		"--implicit-dirs",
```

Comment:
same. why do you have implicit-dirs flag here if you are passing implicit dirs as a flag param.